src/data-types/numeric.ts

```diff
@@ -1,18 +1,19 @@
+import { DataType, ParameterData } from '../data-type';
 const NumericN = require('./numericn');
 
-module.exports = {
+const Numeric: & DataType = {
```

Comment:
Is this a typo? 🤔

Reply:
(The '&' is unnecessary)

Reply:
oh, I will remove the &. I think I have few other &s need to be removed.